web/app/themes/mitlib-parent/css/scss/partials/_home.scss

```diff
@@ -40,197 +50,242 @@ body.page-home {
 			}
 		}
 	}
+
 	//
 	// Hours & Locations
 	//
 	.hours-locations {
 		display: flex;
 		flex-direction: column;
+
 		h2 {
-			border-bottom: 1px solid $color-secondary;
```

Comment:
This change will definitely allow us to confirm the workflow around deploying updates to the compiled stylesheets.